src/reflect/scala/reflect/internal/Positions.scala

```diff
@@ -102,31 +102,34 @@ trait Positions extends api.Positions { self: SymbolTable =>
       throw new ValidateException(msg)
     }
 
-    def validate(tree: Tree, encltree: Tree): Unit = {
+    def validate(tree: Tree, encltree: Tree, trace: Boolean): Unit = {
```

Comment:
need to find a better name for ```trace``` on first reading I was struggling to work out what it was for.

Reply:
I think this comment is invalid - I misread the code on the first pass.